src/aleph/vm/orchestrator/chain.py

```diff
@@ -0,0 +1,59 @@
+import logging
+from typing import Dict, Optional, Union
+
+from aleph_message.models import Chain
+from pydantic import BaseModel
+
+logger = logging.getLogger(__name__)
+
+
+class ChainInfo(BaseModel):
+    """
+    A chain information.
+    """
+
+    chain_id: int
+    rpc: str
+    token: str
+    super_token: Optional[str] = None
+    testnet: bool = False
+    active: bool = True
+
+
+STREAM_CHAINS: Dict[Union[Chain, str], ChainInfo] = {
+    # TESTNETS
+    "SEPOLIA": ChainInfo(
+        chain_id=11155111,
+        rpc="https://eth-sepolia.public.blastapi.io",
+        token="0xc4bf5cbdabe595361438f8c6a187bdc330539c60",
+        super_token="0x22064a21fee226d8ffb8818e7627d5ff6d0fc33a",
+        active=False,
+        testnet=True,
+    ),
+    # MAINNETS
+    Chain.ETH: ChainInfo(
+        chain_id=1,
+        rpc="https://eth-mainnet.public.blastapi.io",
+        token="0x27702a26126e0B3702af63Ee09aC4d1A084EF628",
+        active=False,
+    ),
+    Chain.AVAX: ChainInfo(
+        chain_id=43114,
+        rpc="https://api.avax.network/ext/bc/C/rpc",
+        token="0xc0Fbc4967259786C743361a5885ef49380473dCF",
+        super_token="0xc0Fbc4967259786C743361a5885ef49380473dCF",
+    ),
+    Chain.BASE: ChainInfo(
+        chain_id=8453,
+        rpc="https://base-mainnet.public.blastapi.io",
+        token="0xc0Fbc4967259786C743361a5885ef49380473dCF",
+        super_token="0xc0Fbc4967259786C743361a5885ef49380473dCF",
+    ),
+}
+
+
+def get_chain(chain: str) -> ChainInfo:
+    if chain in STREAM_CHAINS.keys():
+        return STREAM_CHAINS[chain]
+    else:
+        raise ValueError(f"Unknown chain id for chain {chain}")
```

Comment:
What about [EAFP (Easier to Ask for Forgiveness than Permission) here](https://stackoverflow.com/a/11360880) ? Why is a `ValueError` better than a `KeyError` ? 
```suggestion
def get_chain(chain: str) -> ChainInfo:
    try:
        return STREAM_CHAINS[chain]
    except KeyError as error:
        raise ValueError(f"Unknown chain id for chain {chain}") from error
```